woocommerce/woocommerce
# Update product editor package

### All Submissions:






### Changes proposed in this Pull Request:

This PR copies over the Slot fill components for the product screen from the @woocommerce/components package to the @woocommerce/product-editor package. I have added deprecated messages to them within the components package.

Closes #36355  .






### How to test the changes in this Pull Request:




1. Build this branch and go to **Products > Add New** with the new product management feature enabled
2. Open your console and make sure there are no `deprecated` warnings showing and everything still works as expected.
3. Check the above also in the variations view (**Options > edit**)
4. Create a JS file from [this gist](https://gist.github.com/louwie17/7564bb05cce17570fd8bba2d8734da97) and import it. Go to the new Test tab and make sure the deprecated warning shows up, but things still render correctly.



### Other information:




### FOR PR REVIEWER ONLY:
